silta-cicd/circleci-php7.2-node14-composer1/Dockerfile

```diff
@@ -28,10 +28,16 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.
   && sudo apt-get clean
 
 # Install AWS cli and aws-iam-authenticator
-RUN sudo apt install -y awscli git python3 \
-  && curl -o /tmp/aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.18.9/2020-11-02/bin/linux/amd64/aws-iam-authenticator \
-  && chmod +x /tmp/aws-iam-authenticator \ 
-  && sudo mv /tmp/aws-iam-authenticator /bin/aws-iam-authenticator
+RUN sudo apt install -y git python3
```

Comment:
It's unnecessary to have the installation of git and python3 in a separate RUN line, please move them as part of the same RUN line as below, which is also installing unzip and curl.

Having it separate creates an extra image layer.

Reply:
Grouped up package installs and increased image version